app/src/main/java/com/cmput301w23t09/qrhunter/leaderboard/LeaderboardFragment.java

```diff
@@ -187,4 +193,22 @@ private void renderLeaderboards(List<Leaderboard<?>> leaderboards) {
       leaderboardAdapter.notifyDataSetChanged();
     }
   }
+
+  /** Sets up the search view to respond to user input */
+  private void setUpPlayerSearch() {
+    playerSearchView.setOnQueryTextListener(
+        new SearchView.OnQueryTextListener() {
+          @Override
+          public boolean onQueryTextSubmit(String query) {
+            Toast.makeText(getContext(), query, Toast.LENGTH_SHORT).show();
```

Comment:
any particular reason for this?

Reply:
That was originally just meant for debugging purposes but I forgot to remove it. Thanks for pointing it out William!